packages/table/README.md

```diff
@@ -329,6 +300,207 @@ const renderItem = (item: Item, index: number): TemplateResult => {
 
 Please note that there is a bug when attempting to select all virtualised elements. The items are selected programatically, it's just not reflected visually.
 
+### Selection
+
+When making a selection on a virtualized table, it can sometimes be useful to track that selection as something other than an array of indexes. To make this possible the `itemValue` property will accept a method who argument is an item object, return a computed string value to track selection with this value rather than the item's index:
```

Comment:
```suggestion
When making a selection on a virtualized table, it can sometimes be useful to track that selection as something other than an array of indexes. To make this possible, the `itemValue` property accepts a method whose argument is an`Item` object. It then returns a computed string value to track selection with this value instead of the item's index:
```

I think this wording is clearer. Not sure about the change from item to `Item`, but I wanted to check if that's what you meant. In that case, this styling provides more clarity for me.